extension/chrome/elements/compose-modules/compose-send-btn-module.ts

```diff
@@ -214,6 +214,9 @@ export class ComposeSendBtnModule extends ViewModule<ComposeView> {
         throw e;
       }
     }
+    if (msg.externalId) {
+      this.view.acctServer.messageGatewayUpdate(msg.externalId, msgSentRes.id).catch(Catch.reportErr);
```

Comment:
I think we can `await` this, and put it just after `draftDelete`. I suppose we could make the two calls in parallel if we want to speed it up a little bit.